src/AzureRMTools.ts

```diff
@@ -112,7 +112,9 @@ export class AzureRMTools {
         context.subscriptions.push(vscode.window.registerTreeDataProvider("azurerm-vscode-tools.template-outline", jsonOutline));
         context.subscriptions.push(
             // tslint:disable-next-line:typedef
-            vscode.languages.registerCodeActionsProvider(armTemplateLanguageId, new RenameCodeActionProvider(async (document, position) => await this.getPositionContext(document, position, Cancellation.cantCancel)), {
+            vscode.languages.registerCodeActionsProvider(templateOrParameterDocumentSelector,
+                // tslint:disable-next-line:align
+                new RenameCodeActionProvider(async (document, position) => await this.getPositionContext(document, position, Cancellation.cantCancel)), {
```

Comment:
If you start "async" no a new line, format doc will align things properly.

Reply:
Yes, but then I would get an align warning later on that line.
I solved the problem by introducing a new function that did most of the work.